package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "lighthouse-stack-packs",
+  "version": "0.0.1",
+  "description": "Lighthouse Stack Packs",
+  "repository": "GoogleChrome/lighthouse-stack-packs",
+  "keywords": [
+    "google",
+    "chrome",
+    "lighthouse"
+  ],
+  "author": "The Chromium Authors",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/GoogleChrome/lighthouse-stack-packs/issues"
+  },
+  "homepage": "https://github.com/GoogleChrome/lighthouse-stack-packs#readme",
+  "devDependencies": {
+    "typescript": "^3.7.5"
+  },
+  "scripts": {
+    "prepublish": "generate-types",
+    "generate-types": "tsc -p ."
```

Comment:
does there need to be a `types` property in package.json too?

Reply:
types do not work here because we don't export any types for the root file. They are generated on a file basis. Typescript is smart enough to look for a d.ts equivalent.